test/src/test/java/org/corfudb/integration/CorfuReplicationTrimIT.java

```diff
@@ -28,6 +29,13 @@ public void setupPluginPath() {
         }
     }
 
+    @After
+    public void clean() throws Exception {
```

Comment:
the cleanup in the base class forcefully kills all running corfu and lr processes.  Another method here shouldnt be required IMO

Reply:
Thanks, Done.